sonar-orchestrator/pom.xml

```diff
@@ -162,6 +162,14 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- https://mvnrepository.com/artifact/org.postgresql/postgresql -->
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>42.5.0</version>
+  </dependency>
+
+
```

Comment:
@pierre-guillot-sonarsource 
We need to remove this dependency. it brokes the CI tests.